src/uu/kill/src/kill.rs

```diff
@@ -132,13 +132,13 @@ fn table() {
     let mut name_width = 0;
     /* Compute the maximum width of a signal name. */
     for s in &ALL_SIGNALS {
-        if s.name.len() > name_width {
-            name_width = s.name.len()
+        if s.len() > name_width {
+            name_width = s.len()
         }
     }
 
     for (idx, signal) in ALL_SIGNALS.iter().enumerate() {
-        print!("{0: >#2} {1: <#8}", idx + 1, signal.name);
+        print!("{0: >#2} {1: <#8}", idx + 1, signal);
         //TODO: obtain max signal width here
```

Comment:
Outdated comment or is the idea to calculate the width in the same interation?

Reply:
I think this todo referred to the fact that `name_width` that is calculated above is not used here to determine the needed indentation.